docs/finalproject/external-projects.md

```diff
@@ -33,19 +29,14 @@ and to give our partners:
 
 More details can be found on the [Intro](./intro) and [Preparation](./prep) pages.
 
-## What does a Product Owner have to do?
+## We will need a Product Owner of your organisation on the team?
```

Comment:
```suggestion
## We will need a Product Owner from your organisation on the team
```
- A statement is not a question.
- They don't need to own their org product, only this product